ci/install-test-deps.sh

```diff
@@ -16,7 +16,7 @@ case "$OS" in
         yum install -y expect json-glib-devel libtasn1-devel net-tools python3 socat
         ;;
 
-    # NOTE: ubuntu:20.04 uses libtss2-dev provided through the package
+    # NOTE: ubuntu:20.04+ uses libtss2-dev provided through the package
```

Comment:
```suggestion
    # NOTE: ubuntu:20.04 uses libtss2-dev provided through the package
```
I cannot add comments below line 22, but another change (provided 22.04's swtpm module works) should be:
```sh
        'debian:'*|'ubuntu:18.04'|'ubuntu:22.04')
```

Reply:
Since there is a [swtpm package starting with 22.04](https://packages.ubuntu.com/search?keywords=swtpm) any reason we wouldn't install and use it when running the test on 22.04 (and skip building the third-party binaries ourselves)?

Reply:
Oh, I did not know it was packaged for 22.04. We should be able to use the packaged one.

Reply:
Ok, then I'll make some minor adjustments to the script to have 22.04 install and use the package instead of building.

Reply:
Take a look at the latest commit